src/StoreKeeper/WooCommerce/B2C/Tasks/OrderPaymentStatusUpdateTask.php

```diff
@@ -19,7 +19,12 @@ public function run(array $task_options = []): void
             if ('expired' === $paymentData['status']) {
                 $wcOrder = $this->getOrderByStoreKeeperId($storekeeper_id);
 
-                if ($wcOrder && 'cancelled' !== $wcOrder->get_status()) {
+                if ($wcOrder && 'pending' === $wcOrder->get_status()) {
+                    $wcOrder->add_order_note(
+                        sprintf(
+                            __('StoreKeeper: Order was automatically cancelled due to payment expiration (Payment ID=%s)'),
```

Comment:
I think the other messages use TRX not id

Reply:
Updated